src/coreclr/tools/aot/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -1294,11 +1302,13 @@ private MethodWithToken ComputeMethodWithToken(MethodDesc method, ref CORINFO_RE
 
         private ModuleToken HandleToModuleToken(ref CORINFO_RESOLVED_TOKEN pResolvedToken, MethodDesc methodDesc, out object context, ref TypeDesc constrainedType)
         {
-            if (methodDesc != null && (_compilation.NodeFactory.CompilationModuleGroup.VersionsWithMethodBody(methodDesc)
-                || (pResolvedToken.tokenType == CorInfoTokenKind.CORINFO_TOKENKIND_DevirtualizedMethod)
+            if (methodDesc != null && !methodDesc.IsAbstract &&
+                (_compilation.NodeFactory.CompilationModuleGroup.VersionsWithMethodBody(methodDesc) 
+                || (pResolvedToken.tokenType == CorInfoTokenKind.CORINFO_TOKENKIND_DevirtualizedMethod) 
                 || methodDesc.IsPInvoke))
             {
-                if ((CorTokenType)(unchecked((uint)pResolvedToken.token) & 0xFF000000u) == CorTokenType.mdtMethodDef &&
+                CorTokenType corTokenType = (CorTokenType)(unchecked((uint)pResolvedToken.token) & 0xFF000000u);
+                if ((corTokenType == CorTokenType.mdtMethodDef || corTokenType == CorTokenType.mdtMemberRef) &&
```

Comment:
This change will send MANY existing functional scenarios down the specified OwnerType path. I'm not convinced this is wrong, but I'd like to know what prompted this particular change and if we can make this change only for the new static virtuals case.

Reply:
Reverted in a later commit...